docs/mkdocs/docs/api/basic_json/to_bjdata.md

```diff
@@ -34,6 +37,9 @@ The exact mapping and its limitations is described on a [dedicated page](../../f
 
 `use_type` (in)
 :   whether to add type annotations to container types (must be combined with `#!cpp use_size = true`); optional,
+
+`version` (in)
+:   which version of BJData to use (see [draft 3](../../features/binary_formats/bjdata.md#draft-3-binary-format)); optional,
```

Comment:
This new parameter should have gone after the next line, which is the default value for `use_type`.